ada_feeding_perception/ada_feeding_perception/face_detection.py

```diff
@@ -327,11 +325,13 @@ def depth_callback(self, msg: CompressedImage):
                     desired_encoding="passthrough",
                 )
 
-                # Retrieve the depth value averaged over all mouth coordinates
+                # Retrieve the depth value averaged over a 9x9 pixel square around
+                # point between eyes
                 depth_sum = 0
-                for point in img_mouth_points:
-                    depth_sum += closest_depth[int(point[1])][int(point[0])]
-                depth = depth_sum / float(len(img_mouth_points))
+                for x in range(u - 4, u + 4):
+                    for y in range(v - 4, v + 4):
+                        depth_sum += closest_depth[int(x)][int(y)]
```

Comment:
```suggestion
                        depth_sum += closest_depth[int(y)][int(x)]
```

Y and X should be swapped, right?

Reply:
Another suggestion: `int(x)` rounds down. But if the landmark point's position is e.g., `51.99`, that means the detector thought it was closer to `52` than `51`. To account for this I think we should round the float before casting it to an int.